package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -382,6 +382,15 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
         },
 
         /**
+         * Returns the dashboard store on the prompt panel
+         *
+         * @return {Object}
+         */
+        getDashboard: function() {
+          return this.dashboard;
+        },
+
```

Comment:
I am not disagreeing, but i wonder why we try to encapsulate so much inner functionality from the API? If we make the API a "passthrough" to encapsulated functions instead of providing implementations, then why wouldn't we just document and expose the prompt panel object's public functions?